Vizzuality/trase
# Fix/map feedback

## Pivotal Tracker

https://www.pivotaltracker.com/story/show/173708986
https://www.pivotaltracker.com/story/show/173627548

## Description

- Make regions highlight on the map when we hover over the Sankey nodes
- Remove map and logistics map code. We have to remember to be careful merging `feature/new-mapbox-map` branch with develop only when we know the client is ready.
- Make hover and selected in map white and with full opacity
- Clear last hovered or previously selected node on map
- Remove Trase id on tooltips on Indonesia mills (Still waiting for the mill capacity data)
- On the Download page, style the tabs according to the new design (This design is not available on Invision, only on this basecamp thread: https://basecamp.com/1756858/projects/12498794/messages/91940977)

![image](https://user-images.githubusercontent.com/9701591/87514273-44c08980-c67a-11ea-906f-d43c455942d3.png)
![image](https://user-images.githubusercontent.com/9701591/87513823-99afd000-c679-11ea-9556-95ed0d27c773.png)



## Testing instructions

- Hover over municipalities nodes, for example, and the municipalities should highlight on the map
- Hover over a node on the map and out of it. It should remove the highlight. Same if you select and deselect a node.
- The hover and select color should be white
